api/v1alpha1/nifidataflow_types.go

```diff
@@ -33,8 +32,7 @@ type NifiDataflowSpec struct {
 	// contains the reference to the NifiRegistry with the one the dataflow is linked.
 	RegistryClientRef *RegistryClientReference `json:"registryClientRef,omitempty"`
 	// describes the way the operator will deal with data when a dataflow will be updated : drop or drain
-	// +kubebuilder:validation:Enum={"drop","drain"}
```

Comment:
Why this change ?

Reply:
To make this kubebuilder annotation available to all the resources using the PKIBackend type to avoid having to copy and paste.